setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = py_mmd_tools
+version = attr: py_mmd_tools.__version__
+author = MET Norway
+author_email = servicedesk@met.no
+description = Python tools and module for py-mmd-tools
+url = https://github.com/metno/py-mmd-tools
+long_description = file: README.md
+long_description_content_type = text/markdown
+license_file = LICENSE
+license = Apache 2.0
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Atmospheric Science
+python_requires = >=3.6
+project_urls =
+    Source Code = https://github.com/metno/py-mmd-tools
+
+install_requires = ['coverage==5.1', 'filehash', 'ipdb', 'jinja2',
```

Comment:
```suggestion
install_requires = ['filehash', 'jinja2',
```

Reply:
ipdb and coverage are only needed in development

Reply:
can this list be taken from requirements.txt instead? Then we need to split it in two (one for general tools, and an additional one for development tools)... Please make an issue about that, as it is not necessary to do it now.